docs/editor/accessibility.md

```diff
@@ -70,179 +70,181 @@ In `settings.json` file, nest the following code inside the outermost curly brac
 
-The **Go to Next/Previous Difference** actions (`kb(editor.action.accessibleDiffViewer.next)` and `kb(editor.action.accessibleDiffViewer.prev)`), when in a diff editor pane, will bring up the Accessible Diff Viewer, which allows the navigation of the diffs, presented in a unified patch format. Arrow Up and Arrow Down can be used to navigate through the unchanged, inserted, or deleted lines. Pressing `kbstyle(Enter)` will return focus to the modified pane of the diff editor at the selected line number (or closest still existing line number in case a deleted line is selected). Use `kbstyle(Escape)` or `kbstyle(Shift+Escape)` to dismiss the Accessible Diff Viewer.
+When the suggestions pop up, they will get announced to screen readers. Navigate the suggestions using `kbstyle(Ctrl+Up)` and `kbstyle(Ctrl+Down)` and dismiss them with `kbstyle(Shift+Escape)`. If the suggestions get in your way, you can turn them off with the `editor.quickSuggestions` setting.
+
+In a diff editor pane, the **Go to Next/Previous Difference** actions (`kb(editor.action.accessibleDiffViewer.next)` and `kb(editor.action.accessibleDiffViewer.prev)`) will show the Accessible Diff Viewer, presented in a unified patch format. Navigate through the unchanged, inserted, or deleted lines with `kbstyle(Up)` and `kbstyle(Down)`. Press `kbstyle(Enter)` to return focus to the modified pane of the diff editor at the selected line number (or the closest line number that still exists, if a deleted line is selected). Use `kbstyle(Escape)` or `kbstyle(Shift+Escape)` to dismiss the Accessible Diff Viewer.
 
 ## Accessibility help
 
 The command **Open Accessibility Help** `kb(editor.action.accessibilityHelp)` opens a help menu based on the current context. It currently applies to the editor, terminal, notebook, chat panel, and inline chat features.
 
-You can dismiss the accessibility help menu or open additional documentation, if any, from within the help menu.
+You can dismiss any open accessibility help menus or additional help documentation from within the help menu.
 
-![When triggered in an editor, a dialog will appear that explains how to enable screen reader mode explicitly, tab focus mode and other details](images/accessibility/status.png)
+![Help dialog explains how to turn on screen reader mode, tab focus mode, and other details](images/accessibility/status.png)
 
-## Accessible View
+## Accessible view
 
-Run the command **Open Accessible View** `kb(editor.action.accessibleView)` to display an Accessible View to inspect content character by character, line by line. The Accessible View is currently available for hovers, notifications, Jupyter notebook output, and chat responses.
+Run the command **Open Accessible View** `kb(editor.action.accessibleView)` to display an accessible view to inspect content character by character and line by line. The accessible view is currently available for hovers, notifications, Jupyter notebook output, and chat responses.
```

Comment:
The "Accessible View" is an example of a specific VS Code "thing" so we capitalize it.

Reply:
I'd prefer that we leave the sentence as is as we want users to think of the "Accessible View" is a tool that is available for an increasing number of VS Code UI elements. New Accessible Views have been added in the last few releases.